FilePersistence/src/version_control/ListMergeComponent.cpp

```diff
@@ -275,6 +275,12 @@ bool onlyConflictsOnLabel(ConflictPairs& conflictPairs, QSet<Model::NodeIdType>&
 
 		if (changeA && changeB)
 		{
+			if (changeA->onlyLabelChange() && changeB->onlyLabelChange())	continue;
+			else return false;
+			qDebug() << containerId;	//	Used for the code described below
```

Comment:
the proper way to keep an unused variable is `(void) variableName;`. Use that instead of qDebug.